src/views/VolunteersView.vue

```diff
@@ -1,9 +1,24 @@
 <template>
-  <div><h1>Volunteers</h1></div>
+  <div>
+    <h1>Volunteers</h1>
+    <base-button @click.prevent="newVolunteerModal = true">Neue:r Freiwillige:r</base-button>
+    <modal-container v-if="newVolunteerModal">
+      <new-volunteer @close="newVolunteerModal = false" />
```

Comment:
is it the NewVolunteer-component? so: `<NewVolunteer />`  or is this modal specific stuff?